test/examples/row_stripes.test.js

```diff
@@ -11,7 +11,7 @@
 describe("row_stripes.html", () => {
     beforeAll(async () => {
         await page.setViewport({width: 200, height: 100});
-        await page.goto("http://localhost:8081/dist/examples/row_stripes.html");
+        await page.goto("http://localhost:8081/dist/examples/row_stripes.html", {waitUntil: "networkidle0"});
```

Comment:
`yarn test` runs locally even without me adding this `waitUntil`, but this was my attempt to get around

```
    Evaluation failed: TypeError: window.removeStripes is not a function
```